src/formats/subrip/subripinputformat.h

```diff
@@ -48,7 +48,7 @@ class SubRipInputFormat : public InputFormat
 
 			offset += m_regExp.matchedLength();
 
-			QString text(data.mid(offset, (unsigned)m_regExp.indexIn(data, offset) - offset));
+			QString text(data.mid(offset, m_regExp.indexIn(data, offset) - offset));
```

Comment:
This looks to me like a problem, are you sure about it?
If m_regExp.indexIn() doesn't find anything it will return (unsigned)-1, and will be converted to 2^32 - 1, so it will take remaining of the string.
Without the cast it will be come some negative number (-1 - offset) and am not sure what data.mid() will return.
How it was before data.mid(offset, someVeryBigNumber) would return remaining of the string from offset.